website/content/docs/connect/config-entries/jwt-provider.mdx

```diff
@@ -126,8 +150,27 @@ JSONWebKeySet = {
               MaxInterval = "10s"
             }
           }
+          JWKSCluster = {
+            DiscoveryType = "STATIC"
+            ConnectTimeout = "10s"
+            TLSCertificates = {
+              TrustedCA = {                                # cannot specify with JWKSCluster{}.CaCertificateProviderInstance
+                Filename = "<path/to/cert/file>"           # cannot specify with JWKSCluster{}.TrustedCA.EnvironmentVariable, JWKSCluster{}.TrustedCA.InlineString or JWKSCluster{}.TrustedCA.InlineBytes
+                EnvironmentVariable = "<env-variable>"     # cannot specify with JWKSCluster{}.TrustedCA.Filename, JWKSCluster{}.TrustedCA.InlineString or JWKSCluster{}.TrustedCA.InlineBytes
+                InlineString = "<inline-string>"           # cannot specify with JWKSCluster{}.TrustedCA.Filename, JWKSCluster{}.TrustedCA.EnvironmentVariable or JWKSCluster{}.TrustedCA.InlineBytes
+                InlineBytes = "\302\000\302\302\302\302"   # cannot specify with JWKSCluster{}.TrustedCA.Filename, JWKSCluster{}.TrustedCA.InlineString or JWKSCluster{}.TrustedCA.EnvironmentVariable
+              }
+            }
+            TLSCertificates = {
```

Comment:
Are there supposed to be two `TLSCertificates` blocks?

Reply:
yes, just following what we already have in the docs. This seems to be the pattern the doc team is following now

Reply:
i think something we might want to consider is putting them in the same block  and commenting out the second option. Not sure how that could look though

Reply:
I find this "here's the complete configuration" section a little strange since it's re-iterating our actual documentation in an unrealistically verbose config entry. It's intimidating to see such a large code block.

I recall other parts in our docs having specific examples of common use-cases like "file-defined CA example" "external provider example". I consider those opinionated snippets more valuable than "this is everything that can be possibly be done".

Reply:
Thanks for the context! It's definitely non-trivial work to balance exhaustive documentation with simple examples. Really appreciate the feedback process in these PRs 🙏